rpcserver.go

```diff
@@ -9085,5 +9091,10 @@ func (r *rpcServer) getChainSyncInfo() (*chainSyncInfo, error) {
 	// Overwrite isSynced and return.
 	info.isSynced = height == bestHeight
 
+	if !info.isSynced {
+		rpcsLog.Debugf("Blockbeat is not synced to height %v yet",
```

Comment:
So simply logging this slows things down enough for a flake to not occur?

Reply:
nope - it's a rare flake that caused the syncing to be blocked, but there was little logging in that build. I added more debug logs such that if it happens again, we can have more info here.